amundsen_application/static/css/_icons.scss

```diff
@@ -96,6 +96,11 @@ img.icon {
     background-color: transparent;
   }
 
+  &.icon-presto {
+    content: url('/static/images/icons/logo-presto.svg');
```

Comment:
Any reason to use `content` instead of `mask-image` like everything else?

Reply:
Spoke offline. There is a cross-browser incompatibility issue with this css, we have filed a ticket for a future improvement.